userguide/tasks.rst

```diff
@@ -1611,8 +1613,11 @@ An example is shown in
    Cloud Sync Status
 
 
-When an existing task has run, a :literal:`✓` or :literal:`x` is shown
-to reflect the success or failure of the task. Click either symbol to
+When a cloud sync task has run, :literal:`SUCCESS`,
+:literal:`FAILURE`, or :literal:`ABORTED` is shown. :literal:`ABORTED`
+is shown if a cloud sync is stopped while it was running. If a cloud
+sync task has failed, a short description of why it failed is displyed
+after :literal:`FAILURE`. Click the status of the cloud sync task to
```

Comment:
Not clear which button or element is the status. Also, the next three sentences are kind of unclear.
"Click (something) to see the task logs. Click :guilabel:`DOWNLOAD LOGS` to download the log files."